nsdb-kafka-connect/src/main/scala/io/radicalbit/nsdb/connector/kafka/sink/NsdbSinkWriter.scala

```diff
@@ -171,17 +227,10 @@ object NsdbSinkWriter {
     val namespaceField = parsedKcql.namespaceField
     val aliasMap       = parsedKcql.aliasesMap
 
-    require(
```

Comment:
do we still check that db and namespace are defined in kcql or as  connector properties?

Reply:
No, We don't. Let me revert this.